frontend/pages/hosts/details/cards/Queries/_styles.scss

```diff
@@ -1,14 +1,20 @@
 .host-queries-card {
```

Comment:
Can we set a minimum height for this if there's multiple pages? It's just jumpy if the query per page changes on the last page.


https://github.com/user-attachments/assets/2de8dc62-a4da-45bc-9655-8cec23f8db68